@here/harp-mapview/lib/Tile.ts

```diff
@@ -405,8 +407,7 @@ export class Tile implements CachedResource {
     numFramesVisible: number = 0;
 
     /**
-     * The visibility status of the `Tile`. It is actually visible or planned to become visible, and
-     * as such it should not be removed from cache.
+     * The visibility status of the [[Tile]]. It is actually visible or planned to become visible.
      */
     isVisible: boolean = false;
```

Comment:
Why do we still need it? Can we replace it with a getter method that takes the current frame number:
```suggestion
    isVisibleInFrame(frame:number): boolean {
        return frame === frameNumLastRequested;
    }
```

Reply:
I made it a getter, the frame number can be obtained by the datasource's mapview's framenumber (this should be refactored at some point).